Resources/views/standard_layout.html.twig

```diff
@@ -8,18 +8,18 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 
 #}
-{% set _preview              = block('preview') %}
-{% set _form                 = block('form') %}
-{% set _show                 = block('show') %}
-{% set _list_table           = block('list_table') %}
-{% set _list_filters         = block('list_filters') %}
-{% set _tab_menu             = block('tab_menu') %}
-{% set _content              = block('content') %}
-{% set _title                = block('title') %}
-{% set _breadcrumb           = block('breadcrumb') %}
-{% set _actions              = block('actions') %}
-{% set _navbar_title         = block('navbar_title') %}
-{% set _list_filters_actions = block('list_filters_actions') %}
+{% set _preview              = block('preview') is defined ? block('preview') : null %}
```

Comment:
I don't get it… how can a method return something that is not defined? Even `null` is defined… it's… `null`. Not saying this is wrong [the documentation suggests this is valid](http://twig.sensiolabs.org/doc/tests/defined.html), it's just… I don't get what this does. Also, why not use the `default` filter here, like this : `block('preview')|default(null)` ?